_schemaV2.graphql

```diff
@@ -8291,6 +8301,7 @@ type Me implements Node {
   hasPassword: Boolean!
   hasQualifiedCreditCards: Boolean
   hasSecondFactorEnabled: Boolean!
+  icon: Image
```

Comment:
**nitpick:** it's late for this but it would have been great if this was called avatar instead

Reply:
I thought about it. But icon seems to be the preferred word in artsy for when a picture represents an object: Profiles: partners, fairs etc and artists. So I continued with the lingo to avoid confusion